migrations/20171228151840_WpYoastIndexable.php

```diff
@@ -15,17 +15,80 @@ class WpYoastIndexable extends Ruckusing_Migration_Base {
 
 	/**
 	 * Migration up.
+	 *
+	 * @return void
 	 */
-	public function up() {
-		$table_name = $this->get_table_name();
+	public function up()
+	{
+		$this->add_indexable_table();
+		$this->add_indexable_hierarchy_table();
+	}
+
+	/**
+	 * Migration down.
+	 *
+	 * @return void
+	 */
+	public function down()
+	{
+		$this->drop_table( $this->get_indexable_table_name() );
+		$this->drop_table( $this->get_indexable_hierarchy_table_name() );
+	}
+
+	/**
+	 * Creates the indexable table.
+	 */
+	private function add_indexable_table() {
+		$table_name = $this->get_indexable_table_name();
 
 		$indexable_table = $this->create_table( $table_name );
 
-		$indexable_table->column( 'permalink', 'string', [ 'null' => true, 'limit' => 191 ] );
+		$this->add_indexable_columns( $indexable_table );
```

Comment:
I just would also put the creation of the columns in this method. That would make it easier to see what's going on.